R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -26,7 +26,7 @@ sc <- sparkR.init()
 sqlContext <- sparkRSQL.init(sc)
 
 test_that("glm and predict", {
-  training <- createDataFrame(sqlContext, iris)
+  training <- suppressWarnings(createDataFrame(sqlContext, iris))
```

Comment:
right, that's what I mean, add a note in SPARK-11976 to remove these.
alternatively, we could open another JIRA to change all these tests to use a different data frame that doesn't have column names with `.`